core/ecschema-metadata/src/UnitConversion/LinearMap.ts

```diff
@@ -0,0 +1,79 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Constant, Unit } from "../ecschema-metadata";
+import { Float } from "./Float";
+import { isUnit } from "./Helper";
+
+export class LinearMap {
```

Comment:
I don't get the LinearMap name. This is the conversion generated at the end of the process ... maybe go look at what Bill S. calls it in his code ... maybe that is a better name.

Reply:
Where can I find his code?

Reply:
He calls it UnitConversion, so I've changed it to that.